modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/ComposerServiceImpl.java

```diff
@@ -996,6 +996,10 @@ public Job imageToVideo(Attachment sourceImageAttachment, String profileId, doub
           MediaPackageException {
     try {
       final EncodingProfile profile = profileScanner.getProfile(profileId);
+      if (profile == null) {
+        logger.error("Encoding profile {} not found", profileId);
+        throw new MediaPackageException("No encoding set");
```

Comment:
This will lead to a stack trace with a non-helpful message and a separate error log entry. Why not just have a proper error message in the Exception?

```suggestion
        throw new MediaPackageException(String.format("Encoding profile %s not found", profileId));
```